examples/seismic/tti/operators.py

```diff
@@ -399,5 +425,60 @@ def ForwardOperator(model, geometry, space_order=4,
     return Operator(stencils, subs=model.spacing_map, name='ForwardTTI', **kwargs)
 
 
+def AdjointOperator(model, geometry, space_order=4,
+                    save=False, kernel='centered', **kwargs):
```

Comment:
if staggered not supported kernel should not be there. No save

Reply:
or ``raise NotImplementedError``